pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -2697,26 +2697,41 @@ type RuntimeServiceClient interface {
 	// RunPodSandbox creates and starts a pod-level sandbox. Runtimes should ensure
 	// the sandbox is in ready state.
 	RunPodSandbox(ctx context.Context, in *RunPodSandboxRequest, opts ...grpc.CallOption) (*RunPodSandboxResponse, error)
-	// StopPodSandbox stops the running sandbox. If there are any running
-	// containers in the sandbox, they should be forcibly terminated.
+	// StopPodSandbox stops any running process that is part of the sandbox and
+	// reclaims network resources (e.g., IP addresses) allocated to the sandbox.
+	// It is recommended to stop all containers in the sandbox before stopping
+	// the sandbox itself. If there are any running containers in the sandbox,
+	// they should be forcibly terminated.
+	// This call is idempotent, and should return no error if all relevant
+	// resources have already been reclaimed. Kubelet will call StopPodSandbox
+	// as soon as possible to reclaim the resources once it notices that the
+	// sandbox is no longer needed. In some rare cases (which can be
```

Comment:
suggest: 
kubelet will call StopPodSandbox at least once before calling RemovePodSandbox. It will also attempt to reclaim resources eagerly, as soon as a sandbox is torn down. Hence, multiple StopPodSandbox calls are expected.